homeassistant/components/discord/notify.py

```diff
@@ -110,8 +110,6 @@ async def async_send_message(self, message: str, **kwargs: Any) -> None:
                 if image_exists:
                     images.append(image)
 
-        await discord_bot.login(self.token)
-
```

Comment:
`discord_bot` defined above is not referencing the stored instance in Hass data. This breaks the `notify` service.

Reply:
You are right!